### PR TITLE
Widgets endpoint

### PR DESCRIPTION
**Pull request by htharker42**
### What issue is this solving?
Add CreatedDate to User model #272
closes #281

### Any helpful knowledge/context for the reviewer?
Is a re-seeding of the database necessary? YES
Any new dependencies to install? NO
Any special requirements to test? NO

### Please make sure you've attempted to meet the following coding standards
- [X] Code builds successfully
- [X] Code has been tested and does not produce errors
- [X] Code is readable and formatted
- [X] There isn't any unnecessary commented-out code

### CODE REVIEW

File: models/tickets.py [hunk 1]
```diff
@@ -50,9 +50,9 @@ def json(self):
         assignedUserData = UserModel.find_by_id(self.assignedUser)
         assignedUser = "{} {}".format(assignedUserData.firstName, assignedUserData.lastName)
 
-        dateTimeStatusChange = datetime.strptime(self.updated, "%d-%b-%Y (%H:%M)")
+        # dateTimeStatusChange = datetime.strptime(self.updated, "%d-%b-%Y (%H:%M)")
```
Comment (NickSchimek) [inline models/tickets.py]: Do we need this commented out code here?

File: resources/tickets.py [hunk 2]
```diff
@@ -84,17 +82,17 @@ class Tickets(Resource):
     parser.add_argument('issue')
     parser.add_argument('assignedUser')
 
-    @jwt_required
+    # @jwt_required
     def get(self):
         return {'Tickets': [ticket.json() for ticket in TicketModel.query.all()]}
 
-    @jwt_required
+    # @jwt_required
```
Comment (NickSchimek) [inline resources/tickets.py]: Should these be commented out? Or can they be deleted completely?

File: models/tickets.py (hunk 1 above)
Comment (htharker42) [reply models/tickets.py]: nope, it was left over from some of the changes to the DateTime vars I made

File: resources/tickets.py (hunk 2 above)
Comment (htharker42) [reply resources/tickets.py]: doh! thought I remembered to go back and fix these

**Review by Chris-Boe — CHANGES_REQUESTED** [review]
Looks good, just got some merge conflicts and some failing tests (that will probably be fixed when the branch is updated)

**Review by Chris-Boe — CHANGES_REQUESTED** [review]
Looks like we got some tests failing on the branch and the PR https://travis-ci.com/github/codeforpdx/dwellinglybackend/jobs/380639930

**Comment by zfregin** [comment]
It looks like this needs to merge the latest changes from the request-access branch. Those changes enable JSON serialization of the RoleEnum

**Comment by htharker42** [comment]
issues with rebasing causing errors in code
created new branch - re implemented changes

**Pull request closed without merging**